python/encode.py

```diff
@@ -0,0 +1,13 @@
+"""Script to encode a string as Base64"""
+import base64
+import sys
+
+test_string = sys.argv[1]
+
+# Encode string argument as bytes
+test_bytes = test_string.encode("utf-8")
+
+encoded_bytes = base64.b64encode(test_bytes)
```

Comment:
Stylistic thing: I would tend to write this like:
```python
encoded_bytes = base64.b64encode(test_string.encode()).decode()
```
or, if that's too terse, like this:
```python
def encode64(test_string: str) -> str:
  test_bytes = test_string.encode()
  encoded_bytes = base64.b64encode(test_bytes)
  return encoded_bytes.decode()
```

Reason being, I'd try to clearly separate the code that's for encoding a string into a base64 string from the code that acquires the input and sends the output.

Reply:
Cool, will rewrite as the one liner. Thanks for sharing your reasoning. Super helpful.